cmd/clusterctl/client/repository/repository_github_test.go

```diff
@@ -32,7 +30,7 @@ import (
 	clusterctlv1 "sigs.k8s.io/cluster-api/cmd/clusterctl/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/test"
-	"sigs.k8s.io/cluster-api/internal/goproxy"
+	. "sigs.k8s.io/cluster-api/internal/goproxy"
```

Comment:
Let's not use a dot import.

```suggestion
	"sigs.k8s.io/cluster-api/internal/goproxy"
```

Reply:
fixed